chapter10/red/exercise3.pl

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+my @words = map { chomp; $_ } <>;
```

Comment:
While it's usually a pretty good idea, the chomp doesn't seem to really make any difference in this exercise. It would also save you having to include that extra map.